kivy/uix/tabbedpanel.py

```diff
@@ -503,6 +503,10 @@ def switch_to(self, header):
             parent.remove_widget(header_content)
         self.add_widget(header_content)
 
+        if do_scroll:
+            tabs = self._tab_strip
+            tabs.parent.scroll_to(header)
```

Comment:
@akshayaurora what's with the animation? It scrolls as casual ScrollView, doesn't it?